projects/packages/connection/src/sso/class-user-admin.php

```diff
@@ -261,18 +261,20 @@ public function invite_user_to_wpcom() {
 			);
 
 			if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
-				$error        = 'invalid-invite-api-error';
-				$query_params = array(
+				$error_code    = 'invalid-invite-api-error';
+				$error_message = is_wp_error( $response ) ? $response->get_error_message() : wp_remote_retrieve_response_message( $response );
```

Comment:
Note: Returning 500 response and "Internal Server Error" message which is not user-friendly. Need to investigate further.